deploy/kubernetes/base/node_setup/windows/node.yaml

```diff
@@ -0,0 +1,82 @@
+#TODO(#40): Force DaemonSet to not run on master.
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-node-win
+spec:
+  selector:
+    matchLabels:
+      app: gcp-compute-persistent-disk-csi-driver
```

Comment:
We may need linux and windows DaemonSets to use different labels. Otherwise the linux DaemonSet may start managing the windows Pods, and vice versa.

Reply:
updated.